src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public abstract class Figure implements BehaviourFigures {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public Figure() {
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    @Override
+    public void draw() {
+
```

Comment:
Goal of abstract class that we make common state, fields, methods.
If there is no common implementation across all figures, you should make your method as `abstract` , to mark them it should be implemented by subclasses (other figures)